### ISSUE TITLE
Unable to sendMessage with InlineKeyboardButton

### ISSUE THREAD

**Issue by arma7x** [go-telegram-bot-api/telegram-bot-api#109]
Below is my code:

```
keyword := "INLINE QUERY HERE"
markup := bot.NewInlineKeyboardMarkup(
	bot.NewInlineKeyboardRow(
		bot.InlineKeyboardButton{
			Text: "Click Me",
			SwitchInlineQueryCurrentChat: &keyword,
		},
	),
)
msg := bot.NewMessage(chatID, "PLEASE CLICK")
//msg.ReplyMarkup = interface{}(markup)
msg.ReplyMarkup = &markup
tgbot.Send(msg)
```

I'm getting error log 'Bad Request' and empty Message{}. I'm follow this issue [#78](https://github.com/go-telegram-bot-api/telegram-bot-api/issues/78) but still unable to solve this problem. Any suggestion???

**Comment by Syfaro** [comment]
Just to confirm, you are using `github.com/go-telegram-bot-api/telegram-bot-api` for your import, and not the gopkg one, yes?

**Comment by arma7x** [comment]
yes already using github.com/go-telegram-bot-api/telegram-bot-api. Error only occur on SwitchInlineQuery & SwitchInlineQueryCurrentChat only

**Comment by MaksMiles** [comment]
Hi, seems, like i find that issue!
`{"inline_keyboard":[[{"text":"Save","switch_inline_query":"invite","pay":false}]]}` 

`{"ok":false,"error_code":400,"description":"Bad Request: BUTTON_TYPE_INVALID"}`

But, if we use it without *pay*...
`{"inline_keyboard":[[{"text":"Save","switch_inline_query":"invite"}]]}`

`{"ok":true,"result":{"message_id":312,"from":{"id":430037552,"first_name":"***","username":"***"},"chat":{"id":315614612,"first_name":"Maks","last_name":"Miles","username":"MaksMiles","type":"private"},"date":1501533757,"text":"Maks"}}`

**Comment by Syfaro** [comment]
Ah, that was all it was? This was closed by #114 then.
